cdap-data-fabric/src/main/java/io/cdap/cdap/spi/data/nosql/NoSqlStructuredTable.java

```diff
@@ -431,6 +431,21 @@ public void deleteAll(Range keyRange) throws InvalidFieldException, IOException
     }
   }
 
+  @Override
+  public void updateAll(Range keyRange, Collection<Field<?>> fields) throws InvalidFieldException, IOException {
+    LOG.trace("Table {}: UpdateAll with range {}", schema.getTableId(), keyRange);
+    try (CloseableIterator<StructuredRow> iterator = getFilterByRangeIterator(keyRange)) {
```

Comment:
Same, make sure the keyRange is primary keys only and fields are non-key valid fields